base-theme/layouts/partials/taxonomy_header.html

```diff
@@ -0,0 +1,8 @@
+<div class="rounded mb-4">
+  <header>
+    <div class="course-section-title-container">
+      <h1 class="text-uppercase font-weight-bold pb-1 mb-3">{{ replace .Title "_" " " }}</h1>
```

Comment:
@gumaerc We can also replace `_` with `" "` in `.Title` in `content_header.html` file but I was unsure whether we have deliberately added underscores (`_`) in any of the page Titles or not. Hence created this new partial.

But if we are sure that we don't want underscores (`_`) in page title then my suggestion would be to delete this partial and do `{{ replace .Title "_" " " }}` in `content_header.html` for better consistency. 

Perhaps if @pdpinch has any thoughts on having underscores (`_`) in page title?

Reply:
I would go with the separate partial for now, although I do agree that intentional underscores in a page title does seem unlikely.  Instead of simply using `replace` though, you might try using `humanize`: https://gohugo.io/functions/humanize/

Reply:
Can we have a proper title "Learning resource types"   https://gohugo.io/content-management/taxonomies/#add-custom-metadata-to-a-taxonomy-or-term ?

Reply:
@Ferdi has a good point.  We can add "stock content" to any theme in the same way you would add content to a Hugo site, under the `content` folder.  So, we could in theory add a file at `course/content/learning_resource_types/_index.md` and set the `title` property in the front matter there.